### PR TITLE
New captcha argument in START_URL

### LINKED ISSUE #6: Unhandled event: [u'error', u'You are temporarily banned.']
I keep getting an error Unhandled event: [u'error', u'You are temporarily banned.']
Although I don't believe I am actually banned, I can acceess omegle through my browser and there are no issues.

```python
from pyomegle import OmegleClient
from pyomegle import OmegleHandler
import time

h = OmegleHandler(loop=True)
c = OmegleClient(h, wpm=47, lang="en")
c.start()

input_str = "message"

while 1:
    time.sleep(5)
    c.send(input_str)
    c.next()
```

### PR DESCRIPTION
**Pull request by danny-burrows**
As far as I'm able to tell this should fix #6 the temporary ban messages.

### CODE REVIEW

**Comment by danny-burrows** [comment]
These are a few extra changes, I delved into the omegle.js source and found these to be the most important.

- The server list can also be updated to include all of Omegles newer 'front1.omegle.com' servers, from 1 to 32.
- RCS was the old method of bot detection and is replaced by 'caps=recaptcha2'.
- From looking through the Omegle source code they now use an 8 char randID instead of 9, and the possible didgets have also been updated to match their source.

Hope this helps. :)

**Comment by MidnightRedMI** [comment]
Hi @danny-burrows I try to use your file updated for the new captcha, but when launch the script I have this error 
NameError: name 'RECAPTCHA_CHALLENGE_URL' is not defined

I hope that this help (i didn't know Python, I only use this script and today I try to use the version based on python3)

**Comment by danny-burrows** [comment]
Hello @MidnightRedMI, I was unable to recreate this, but as far as I can tell this could be caused by the updated version of the module not being properly installed upon using the script.

While this request is still pending the best way to use this version is to clone my fork, download the files, use CMD or a Terminal to navigate to the directory where the setup.py file is located and use `pip install -e .`. This should ensure that the dependencies and paths are configured properly.

Hope this helps. All the best.

**Comment by MidnightRedMI** [comment]
hello @danny-burrows I try to follow your suggestion and I have this response
`Requirement already satisfied: mechanize in c:\python38\lib\site-packages (from pyomegle==1.7) (0.4.5)
Requirement already satisfied: html5lib>=0.999999999 in c:\python38\lib\site-packages (from mechanize->pyomegle==1.7) (1.1)
Requirement already satisfied: six>=1.9 in c:\python38\lib\site-packages (from html5lib>=0.999999999->mechanize->pyomegle==1.7) (1.15.0)
Requirement already satisfied: webencodings in c:\python38\lib\site-packages (from html5lib>=0.999999999->mechanize->pyomegle==1.7) (0.5.1)
Installing collected packages: pyomegle
  Attempting uninstall: pyomegle
    Found existing installation: pyomegle 1.7
    Uninstalling pyomegle-1.7:
      Successfully uninstalled pyomegle-1.7
  Running setup.py develop for pyomegle
Successfully installed pyomegle`

I think that all is updated with your fork, I use this setup on `__init__.py`:

`from pyomegle import *

h = OmegleHandler(loop=False)            # session loop
c = OmegleClient(h, wpm=40, lang='it')  # 47 words per minute
c.start()

while 1:
    input_str = input('')           # string input

    if input_str.strip() == '/next':
        c.next()                        # new conversation
    elif input_str.strip() == '/exit':
        c.disconnect()                  # disconnect chat session
        break
    else:
        c.send(input_str)               # send string`

but when I launch the script have the same problem:
`url = RECAPTCHA_CHALLENGE_URL % challenge
NameError: name 'RECAPTCHA_CHALLENGE_URL' is not defined`

If I go to omegle.com, have the request of captcha, it's possible that is for this the problem?

Regards, all the best

File: pyomegle/pyomegle.py [hunk 1]
```diff
@@ -87,7 +88,7 @@ def __init__(self, events_handler, rcs=1, firstevents=1, spid='', random_id=None
 
     def _randID(self, length):
         """ Generates a random ID for chat session """
-        return ''.join([random.choice('0123456789ABCDEFGHJKLMNPQRSTUVWXYZ')
+        return ''.join([random.choice('23456789ABCDEFGHJKLMNPQRSTUVWXYZ')
```
Comment (eliascotto) [inline pyomegle/pyomegle.py:91]: Why you removed `01`?

**Review by eliascotto — COMMENTED** [review]
@danny-burrows I've tested the changes and seems to work properly. If you explain the commit I could merge your PR

File: pyomegle/pyomegle.py (hunk 1 above)
Comment (danny-burrows) [reply pyomegle/pyomegle.py:91]: After doing some delving into the Omegle.js source on their website I found that their random ID generation has changed, they no longer use 0 or 1 in their random ID's.

Here's the sectionfrom omegle.js...
https://www.omegle.com/static/omegle.js?649
```javascript
var randID=Cookie.read("randid");
randID&&8===randID.length||(randID=function(){for(var a="23456789ABCDEFGHJKLMNPQRSTUVWXYZ",b="",c=0;8>c;c++){var d=Math.floor(Math.random()*a.length);
```


I thought it best to match their style of doing so. There's a chance that RandomID's including certain sequences of 1's or 0's might be rejected by their systems.

Comment (danny-burrows) [reply pyomegle/pyomegle.py:91]: Apologies for not explaining this in the initial commit, it must have escaped me. But I hope this helps. :)

**Comment by danny-burrows** [comment]
> but when I launch the script have the same problem:
> `url = RECAPTCHA_CHALLENGE_URL % challenge NameError: name 'RECAPTCHA_CHALLENGE_URL' is not defined`

@MidnightRedMI, Hi, sorry, I've had some time to look into it and yes, this is a consequence of the captcha_required method needing an update due to the way Omegle handles captchas being changed. You could update RECAPTCHA_CHALLENGE_URL to self.RECAPTCHA_CHALLENGE_URL in the source and it will be able to access that class attribute. 

I haven't fixed this myself or touched this method in this PR because implementing the new captcha method would be a big enough job that it requires a whole separate issue/PR that I haven't had time to dedicate to just yet. 

I'm sorry for the delay in responses and do hope this helps a little.

All the best, 

Danny.

**Comment by eliascotto** [comment]
@danny-burrows All good. Thanks for the effort!

**Pull request merged**